.github/workflows/os-compatibility-test.yaml

```diff
@@ -15,9 +15,31 @@ on:
       - '**/*.md*'
```

Comment:
This missed removing the paths here.

Reply:
This still stands - not sure if you missed this comment with the other one I added to the same line - these `paths-ignore:` should be removed, right? We don't want github doing the filtering, as it is now done by dorny/paths-filter?

Reply:
Thanks! I did indeed miss this. You're right, these should be removed. I'll do that tomorrow.